assets/css/global.less

```
@@ -1014,51 +1012,128 @@ body .guides .guides-section-white {
-  letter-spacing: 0.51px;
-  text-align: center;
-  color: @guides-primary-color;
-  &:hover, &:focus, &:active{
-    .no-transform;
-    background: rgba(0, 185, 248, 0.1);
+#tags-filter {
+  display: flex;
+  justify-content: flex-end;
+  button {
+    border-radius: 8px;
+    background: rgba(0, 185, 248, 0.2);
+    padding: 0 15px;
+    box-shadow: none;
+    border: none;
+    font-size: 18px;
+    font-weight: 600;
+    letter-spacing: 0.51px;
+    text-align: center;
     color: @guides-primary-color;
+    &:hover, &:focus, &:active{
+      .no-transform;
+      background: rgba(0, 185, 248, 0.1);
+      color: @guides-primary-color;
+    }
   }
 }
 
 .filter-options {
   @media (max-width: @screen-md-min) {
     margin-top: 10%;
   }
-  margin-top: 10%;
+  margin-top: 11%;
   padding: 0;
   .card {
-    padding: 1em 4em;
+    padding: 30px 109px;
     border-radius: 8px;
     background-color: #f5f9fa;
+    min-height: 193px;
+    max-height: 193px;
     .tags {
       display: flex;
       flex-wrap: wrap;
-      flex-direction: row;
-    }
+      flex-direction: column;
+      max-height: 133px;
+    }    
     .checkbox {
-      color: black;
-      margin-right: 15px;
-      & input[type="checkbox"]{
-        opacity: 1;
-        width: 18px;
-        height: 18px;
+      padding-left: 20px;
+      position: relative;
+      display: block;
+      margin-bottom: 0;
+      input {
+        opacity: 0;
+        position: absolute;
+        z-index: 1;
+        cursor: pointer;
+        margin-left: -20px;
+        &:checked {
+          + label {
+            &::before {
+              border-color:@guides-primary-color;
+            }
+            &::after {
+              content: '';
+              display: inline-block;
+              position: absolute;
+              width: 13px;
+              height: 13px;
+              left: 2px;
+              top: 4px;
+              margin-left: -20px;
+              border: 1px solid @guides-primary-color;
+              border-radius: 2px;
+              background-color: @guides-primary-color;
+            }
+          }
+        }
+      }
+      label {
+        display: inline-block;
+        position: relative;
+        padding-left: 10px;
+        &::before {
+          content: '';
+          display: inline-block;
+          position: absolute;
+          width: 18px;
+          height: 18px;
+          border-radius: 2px;
+          background-color: #fff;
+        }
       }
     }
-    .checkbox+.checkbox {
+    .checkbox+.checkbox {	
       margin-top: 10px !important;
     }
+    .checkbox {
+      input {
+        &:checked {
+          + label {
+            &::before {
+              background-color: @guides-primary-color;
+            }
+            &::after {
+              content: '';
+              position: absolute;
+              width: 10px;
+              height: 7px;
+              background: @guides-primary-color;
+              left: 3px;
```

```suggestion
              left: 4px;
```
Just a bit of a nitpick for better alignment!